frontend/views/components/group-creation-steps/GroupName.vue

```diff
@@ -51,11 +50,51 @@ export default {
   components: {
     Avatar
   },
+  watch: {
+    'groupInitials': function (initials) {
+      this.updatePictureCanvas(initials)
+    }
+  },
   mounted () {
     this.$refs.name.focus()
+    const c = this.$refs.pictureCanvas
+    c.width = 256
+    c.height = 256
+
+    // Recover the saved picture, in case user is coming back to this step.
+    if (this.$assistant.ephemeral.groupPictureType === 'image') {
+      this.$refs.pictureAvatar.setFromBlob(this.$assistant.ephemeral.groupPictureFile)
+    } else {
+      this.updatePictureCanvas(this.groupInitials)
+    }
+  },
+  beforeDestroy () {
+    if (!this.$assistant.ephemeral.groupPictureFile) {
+      const pictureBase64 = this.$refs.pictureCanvas.toDataURL('image/png')
+      this.$v.form.groupPicture.$touch()
+      this.$assistant.ephemeral.groupPictureFile = b64toBlob(pictureBase64)
```

Comment:
Here is an example of where naming the function correctly would have caught the bug, since `pictureBase64` is a dataURI that uses `image/png` and `b64toBlob` only handles creating `Blob` objects that are of `{type: 'image/jpeg'}`.

Even if it mysteriously still works, it might not work in all browsers/environments, so we should make sure that the encoding type is the same as the data source type.